src/bosh-monitor/lib/bosh/monitor/metric.rb

```diff
@@ -12,5 +12,14 @@ def initialize(name, value, timestamp, tags)
       @timestamp = timestamp
       @tags = tags
     end
+
+    def to_hash
+      {
+          :name => @name,
+          :value => @value.to_s,
```

Comment:
Only one of the values are ints. It was nice to keep it consistent, especially when parsing the values for a golang struct.